### PR TITLE
add support -H option

### PR DESCRIPTION
**Pull request by lupin012**

### CODE REVIEW

**Comment by lupin012** [comment]
Add  option -H to permit to run integration tests with daemons located on remote machine

File: tests/integration/run_tests.py [hunk 1]
```diff
@@ -176,7 +176,7 @@ def run_shell_command(command: str, command1: str, expected_response: str, verbo
     return 0
 
 def run_tests(test_dir: str, output_dir: str, json_file: str, verbose: bool, daemon_under_test: str, exit_on_fail: bool, verify_with_daemon: bool, daemon_as_reference: str,
-              dump_output: bool, test_number, infura_url: str):
+              dump_output: bool, test_number, infura_url: str, daemon_url: str):
```
Comment (canepat) [inline tests/integration/run_tests.py:179]: `daemon_url` -> `daemon_host` according to the meaning of `host` parameter in `get_target` function

File: tests/integration/run_tests.py [hunk 2]
```diff
@@ -253,13 +253,14 @@ def usage(argv):
...
     print("-t <test_number>: run single test")
-    print("-d provides request also to the reference daemon (default RPCDAEMON)")
+    print("-d provides same request also to the reference daemon (default RPCDAEMON)")
     print("-i provides request also to the reference daemon (INFURA)")
     print("-b blockchain (default goerly)")
     print("-v verbose")
     print("-o dump response")
     print("-x exclude api list (i.e txpool_content,txpool_status")
     print("-X exclude test list (i.e 18,22")
+    print("-H IP address where the daemnon are localted(i.e 10.10.2.3) ")
```
Comment (canepat) [inline tests/integration/run_tests.py:263]: Replace with:

"-H host where the daemon is located (i.e 10.10.2.3)"

File: tests/integration/run_tests.py [hunk 3]
```diff
@@ -276,6 +277,7 @@ def main(argv):
     req_test = -1
     dump_output = 0
     infura_url = ""
+    daemon_url = "localhost"
```
Comment (canepat) [inline tests/integration/run_tests.py:280]: `daemon_url` -> `daemon_host`

**Review by canepat — CHANGES_REQUESTED** [review]

File: tests/integration/run_tests.py (hunk 1 above)
Comment (lupin012) [reply tests/integration/run_tests.py:179]: ok renamed deamon_on_host

File: tests/integration/run_tests.py (hunk 2 above)
Comment (lupin012) [reply tests/integration/run_tests.py:263]: ok

File: tests/integration/run_tests.py (hunk 3 above)
Comment (lupin012) [reply tests/integration/run_tests.py:280]: renamed: deamon_on_host

**Review by canepat — APPROVED** [review]

**Pull request merged**